src/sentry/static/sentry/app/views/settings/organizationFiltersAndSampling/rules/column.tsx

```diff
@@ -0,0 +1,8 @@
+import styled from '@emotion/styled';
+
+const Column = styled('div')`
+  display: flex;
+  align-items: center;
+`;
+
+export default Column;
```

Comment:
Should this and `centeredColumn` be co-located in a `styles` module?

Reply:
I used to do that, but I think I prefer to have a separate component.